src/api.rs

```diff
@@ -459,11 +491,13 @@ impl Config {
 
     Ok(())
   }
-
+  /// Create a new context for the encoder. Context contatins
```

Comment:
The context is the full encoder state.

Please omit implementation details that may and will change and mention what the context is useful for :)